app/src/test/java/io/gnosis/safe/ui/splash/SplashViewModelTest.kt

```diff
@@ -119,9 +126,12 @@ class SplashViewModelTest {
         coEvery { notificationRepository.checkPermissions() } returns true
```

Comment:
better to use verifySequence here and also check that invocations happen in the correct order